ballerina-tests/Ballerina.toml

```diff
@@ -1,24 +1,24 @@
 [package]
 org = "ballerina"
 name = "grpc_tests"
-version = "@toml.version@"
+version = "1.3.1"
```

Comment:
is this intentional?

Reply:
This was a mistake. Fixed in https://github.com/ballerina-platform/module-ballerina-grpc/pull/892/commits/4c9e1de2aabe728a957ef6ef65e5e81aeaeada3a